### ISSUE TITLE
ReferenceError: Sandbox is not defined

### ISSUE THREAD

**Issue by xurei** [auth0/sandboxjs#18]
Version 3.4.0 seems to be broken. When using wt-cli, I have this error :
```javacsript
Sandbox.prototype.updateStorage = function (storage, options, cb) {
^

ReferenceError: Sandbox is not defined
    at Object.<anonymous> (/usr/local/lib/node_modules/wt-cli/node_modules/sandboxjs/lib/webtask.js:275:1)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/wt-cli/node_modules/sandboxjs/lib/sandbox.js:8:15)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/wt-cli/node_modules/sandboxjs/lib/index.js:3:15)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
```

**Comment by rickster13** [comment]
Yes we are experiencing the same issue. It gets resolved when I use `Webtask.prototype.getStorage` instead of the `Sandbox.prototype.getStorage`.

**Comment by NotMyself** [comment]
Hi @rickster13 & @xurei,

It looks like this issue was resolved by @jcenturion in version `3.4.1`. Are you still having this issue? I am going to close this issue. If you are still experiencing issues with it, please feel free to respond and reopen it.

Thanks for using webtasks!
